tasks/utils/config.ts

```diff
@@ -32,7 +32,7 @@ const configSchema = z.object({
   NEXT_PUBLIC_PLAYSTORE_LINK: z.string().url().default('https://play.google.com/store/apps'),
   NEXT_PUBLIC_APPSTORE_LINK: z.string().url().default('https://www.apple.com/de/app-store/'),
   NEXT_PUBLIC_OAUTH_ISSUER_URL: z.string().url().default('https://auth.helpwave.de'),
-  NEXT_PUBLIC_OAUTH_REDIRECT_URI: z.string().url().default('https://staging-tasks.helpwave.de/auth/callback'),
+  NEXT_PUBLIC_OAUTH_REDIRECT_URI: z.string().url().default('http://localhost:3000/auth/callback'),
```

Comment:
This is okay but maybe we should provide an addtional `tasks/.env.local` containing this setting.

Reply:
Okay, the defaults should be "production" ready. The `.env` defines the default for the local development setup. The docker build gets not influenced by the `.env` because of the `.dockerignore`. Right now, the only way to influence the build envs are `.env.local`.